software/otterwiki.yml

```
@@ -0,0 +1,12 @@
+name: "An Otter Wiki"
+website_url: "https://github.com/redimp/otterwiki"
+source_code_url: "https://github.com/redimp/otterwiki"
+description: "Simple, easy to use Wiki software using Markdown."
+licenses:
+  - MIT
+platforms:
+  - Python
+  - Docker
+tags:
+  - Wiki
+depends_3rdparty: no
```

Fixes tags
Depends on 3rd party is optional so this can be removed.
```suggestion
tags:
  - Wikis
```